.github/workflows/formatting-checker.yaml

```diff
@@ -0,0 +1,24 @@
+name: "Formating checker"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  HTML-Tidy:
+    runs-on: [ubuntu-latest]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install HTML Tidy
+      run: sudo apt install tidy
+
+    - name: Run HTML Tidy
+      run: tidy -config tidy.cfg -e index.html
```

Comment:
Do we need a non-empty config file?

Reply:
Oops. I meant for this branch to depend on https://github.com/w3c/PNG-spec/pull/229
That other pull request adds the config file.